### DISCUSSION TITLE
How to setup analytics id ?

### DISCUSSION THREAD

**Discussion by edgarriba** [pradyunsg/furo#243] [Questions] [answered]
hi  ! We recently adopted `furo` for our docs at [Kornia](https://kornia.readthedocs.io/en/latest/) - it awesome !

During the port we are missing to include the `analytics_id` in the `html_theme_options` as we did with read the docs template. What's the way to do it in `fure` ?

thanks in advance!

**Comment by pradyunsg** [comment, marked as answer]
Furo doesn't contain any baked-in user tracking or analytics support and this is intentional on my part -- both from a philosophical front of "don't make it easier for Google to track people everywhere" and also from a technical standpoint of various users wanting support for various analytics plaforms.

If you wish to include that in your documentation, you can injecting the relevant code yourself. The process for injecting user-tracking logic into Furo-based documentation is the same as injecting any other JS/CSS/HTML (depending on what your user tracking mechanism is based on) -- the process is documented at https://pradyunsg.me/furo/customisation/injecting/.

**Reply by edgarriba** [reply]
thanks so much - will look into that !
